website/docs/registry/providers/index.mdx

```diff
@@ -28,6 +28,6 @@ Terraform providers are published and maintained by a variety of sources, includ
 <ProviderTable />
 <p></p>
 
-## Verified Provider Development Program
+## Partner Provider Development Program
 
-If your organization is interested in joining our Provider Development Program (which sets the standards for publishing providers and modules with a `Verified` badge), please take a look at our [Program Details](/docs/partnerships) for further information.
+If your organization is interested in joining our Provider Development Program (which sets the standards for publishing providers and modules with a `Partner` badge), please take a look at our [Program Details](/docs/partnerships) for further information.
```

Comment:
```suggestion
If your organization wants to join our Provider Development Program (which sets the standards for publishing providers and modules with a `Partner` badge), please take a look at our [Program Details](/docs/partnerships) for further information.
```

Suggest changing "is interested in joining" to "wants to join" to avoid passive voice style issues.